<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseDocumentWidget</class>
 <widget class="QWidget" name="ChooseDocumentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyWebNCL - Selecionando a Apresentação NCL</string>
  </property>
  <widget class="QPushButton" name="btn_choose_file">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/folder_open.png</normaloff>:/i/folder_open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_document_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="ckb_remote_control">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Controle Remoto</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tamanho:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_size">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>400x300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>480x360</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>800x600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024x768</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280x720</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920x1080</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>90</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar Apresentação</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/button_blue_play.png</normaloff>:/i/button_blue_play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
